driver-mapping/src/main/java/com/datastax/driver/mapping/Mapper.java

```diff
@@ -861,7 +865,7 @@ public void resetDefaultDeleteOptions() {
      */
     public static abstract class Option {
 
-        enum Type {TTL, TIMESTAMP, CL, TRACING, SAVE_NULL_FIELDS, IF_NOT_EXISTS}
+        enum Type {TTL, TIMESTAMP, CL, TRACING, SAVE_NULL_FIELDS, SAVE_NULL_FIELDS_AS_UNSET, IF_NOT_EXISTS}
```

Comment:
It looks like SAVE_NULL_FIELDS_AS_UNSET is no longer used so this can be removed